File: .tmp_py4013/batch_04_sources/2498_frog_jump_ii.js
// LeetCode 2498 - Frog Jump II
// https://leetcode.com/problems/frog-jump-ii/

/**
 * @param {number[]} stones
 * @return {number}
 */
var maxJump = function(stones) {
    let ans = stones[1] - stones[0];
    for (let i = 2; i < stones.length; i++) {
        const diff = stones[i] - stones[i - 2];
        if (diff > ans) ans = diff;
    }
    return ans;
};

File: .tmp_rs4013/rust_check_batch08/2498_frog_jump_ii.rs
struct Solution;
// LeetCode 2498 - Frog Jump II
// https://leetcode.com/problems/frog-jump-ii/

impl Solution {
    pub fn max_jump(stones: Vec<i32>) -> i32 {
        let mut ans = stones[1] - stones[0];
        for i in 2..stones.len() {
            let diff = stones[i] - stones[i - 2];
            if diff > ans {
                ans = diff;
            }
        }
        ans
    }
}

fn main() {}
